explorer/info.go

```diff
@@ -0,0 +1,82 @@
+package explorer
+
+import (
+	"time"
+
+	"go.sia.tech/core/types"
+)
+
+// BlockFacts returns a bunch of statistics about the consensus set as they
+// were at a specific block.
+type BlockFacts struct {
```

Comment:
It'll probably take a few iterations before we settle on the right data to store. I've asked Hakkane (of SiaStats) what kind of API he'd like to see in the new explorer, but in the meantime we can at least be sure about storing the full block and its full validation context.